src/aws_encryption_sdk/internal/formatting/serialize.py

```diff
@@ -257,13 +314,22 @@ def serialize_header_auth(version, algorithm, header, data_encryption_key, signe
     :param bytes data_encryption_key: Data key with which to encrypt message
     :param signer: Cryptographic signer object (optional)
     :type signer: aws_encryption_sdk.Signer
+    :param required_encryption_context_bytes: Serialized encryption context items
+        for all items whose keys are in the required_encryption_context list.
+        This is ONLY processed if using the aws-cryptographic-materialproviders library
+        AND its required encryption context CMM. (optional)
```

Comment:
Maybe I'm not following, but the Default CMM should do this. Getting required EC only happens with the required EC CMM _on encrypt_.

Reply:
Fixed; this was already the case in the code, but not in the documentation.